lib/elixir/lib/module/types/unify.ex

```diff
@@ -788,6 +823,74 @@ defmodule Module.Types.Unify do
     {type, context}
   end
 
+  # TODO: Figure out function expansion
+  # TODO: Flatten across type variables
+
+  def expand_union({:union, types}) do
```

Comment:
expand_union or flatten_union? 🤔

Reply:
Ping. Any way is fine. Just making sure you saw this. :)

Reply:
I have no strong opinion on it but it seems we use flatten in documentation and I think we use it when talking about it so I will change to that to be consistent.